tests/test_output_formatter.cpp

```diff
@@ -276,1056 +295,697 @@ class LocaleSettingTest : public testing::Test
     std::locale saved_locale;
 };
 
-class TableFormatter : public LocaleSettingTest
-{
-};
-
-class JsonFormatter : public LocaleSettingTest
-{
-};
-
-class CSVFormatter : public LocaleSettingTest
-{
-};
+typedef std::tuple<const mp::Formatter*, const ::google::protobuf::Message*, std::string, std::string>
```

Comment:
It's a bit difficult to track what each element means (here and in the petenv suite). Could we maybe have a comment to help the reader?

Reply:
Done.